tests/strictequal.test.js

```diff
@@ -0,0 +1,23 @@
+var describe = require ('mocha').describe
+var it = require('mocha').it
+var trueequal = require('../equal')
+var expect = require('chai').expect
+
+describe('equal', () => {
+    describe('trueequal',() => {
+        it('return true if the value strictly equal to each other',) =>{
+            // setup test data
+            var testData = [2,3,4]
+
+            // call the function from our outside file
```

Comment:
here is an example of a proper test

describe('strictEqual', () => {
  it('return a value of true, if numbers are strictly equal to one another', () => {
    var a = 5
    var b = 5

    let result = strictEqual(a, b)

    expect(result).to.equal(true)
  })